### PR TITLE
Fix BBE related to Lock statement

### PR DESCRIPTION
**Pull request by dulvinw**
Fix command output of Lock BBE

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/locks/locks.out [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # To run this sample, navigate to the directory that contains the
 # `.bal` file, and execute the `ballerina run` command.
```
Comment (praneesha) [inline examples/locks/locks.out]:
```suggestion
# `.bal` file and execute the `ballerina run` command.
```

**Comment by hasithaa** [comment]
@keizer619 Should we send this to stage branch? This is required for 1.2.0 docs.

File: examples/index.json [hunk 2]
```diff
@@ -1072,7 +1072,7 @@
     {
         "title": "Lock",
         "column": 0,
```
Comment (MaryamZi) [inline examples/index.json]: I think this should go to column 2 after the Concurrency examples maybe?

Comment (dulvinw) [reply examples/index.json]: will fix

**Pull request merged**